src/api/layers/wms-capabilities.class.js

```diff
@@ -0,0 +1,174 @@
+import { LayerAttribution } from '@/api/layers/AbstractLayer.class'
+import ExternalWMSLayer from '@/api/layers/ExternalWMSLayer.class'
+import ExternalGroupOfLayers from '@/api/layers/ExternalGroupOfLayers.class'
+import { WMSCapabilities as OlWmsCapabilities } from 'ol/format'
+import log from '@/utils/logging'
+
+/** Wrapper around the OpenLayer WMSCapabilities to add more functionalities */
+export default class WmsCapabilities {
```

Comment:
Maybe a different name would help differentiate them?
Something like `ExtendedWMSCapabilitesParser` or something similar

Reply:
@pakb I'm not a big fan for such long name, but I could use `WmsCapabilitiesParser` ?

Reply:
Done as `WMSCapabilitiesParser`